ArcadiaParties-SPA/src/app/services/auth.service.ts

```diff
@@ -38,11 +37,7 @@ export class AuthService {
   acquireTokenResilient(): Observable<any> {
     return new Observable<any>((subscriber: Subscriber<any>) =>
       this.context.acquireToken('7550c576-a91a-4bd6-8f47-5af6dc701a40', (message: string, token: string) => {
```

Comment:
`message` parameter is not used. In such cases it is recommended to use `_` instead of meaning variable name to let TS know that all code style rules shouldn't be applied.

Reply:
I've already done it before your comment, but confused branches to push. Already fixed